libs/components/modals/src/lib/modules/modal/modal.interface.ts

```diff
@@ -62,6 +62,7 @@ export interface SkyModalConfigurationInterface {
   tiledBody?: boolean;
 
   /**
+   * @deprecated Use SkyHelpInlineComponent in the modal header instead.
```

Comment:
I don't think we want to call out `SkyHelpInlineComponent` specifically. (It's a viable option, but it's not the only option a consumer can take to add inline help. They can use any element so long as it has the `.sky-control-help` class on it.) Take a look at the `SkyInputBoxComponent` JSDoc comment to see how we can word this to be more generic.